pyseir/cli.py

```diff
@@ -56,14 +57,37 @@ def _generate_infection_rate_metric(regions: List[infer_rt.RegionalInput]) -> pd
     Apply infer_rt.run_rt_for_fips for each region in regions and return a combined results table
     with a unique row for each region+date combination.
     """
+    if not regions:
+        return pd.DataFrame()
+
     with Pool(maxtasksperchild=10) as p:
-        results = p.map(infer_rt.run_rt, regions)
+        results = pd.concat(p.map(infer_rt.run_rt, regions))
 
-    if results:
-        return pd.concat(results)
-    else:
+    if results.empty:
         return pd.DataFrame()
 
+    need_patch = set(results.fips) & set(NEW_ORLEANS_FIPS)
+    if need_patch:
+        logging.info("Applying New Orleans Patch")
+        unpatched = results.loc[~results.fips.isin(need_patch), :]
+
+        # Aggregate the results created so far into one timeseries of metrics in a DataFrame
+        nola_rt_inferences = pyseir.rt.patches.patch_aggregate_rt_results(need_patch)
```

Comment:
Also, can we call in `nola_combined_infection_rate` or something that doesn't use the word inferences. I think we have an over-burdened definition for `inference`.

Reply:
Done